docs/install/install-cave-beta.md

```diff
@@ -0,0 +1,165 @@
+# Install CAVE - BETA Version
```

Comment:
I think the name of this file should be `v20` or something...because, likely, we may have other "betas" in the future, so this name would become confusing or duplicative....